jsf/subsystem/src/test/java/org/jboss/as/jsf/deployment/JSFDepProcessorTestCase.java

```diff
@@ -0,0 +1,57 @@
+package org.jboss.as.jsf.deployment;
```

Comment:
@moulalis I think that should have a licence header

Reply:
@spyrkob I've added header.